includes/class-admin-settings.php

```diff
@@ -516,10 +546,10 @@ public function add_settings_field_callback( $args = array() ) {
 				</select>
 				<p>
 					<input
-						type="text" 
-						id="aspireupdate-settings-field-<?php echo esc_attr( $id ); ?>_other" 
-						name="<?php echo esc_attr( $this->option_name ); ?>[<?php echo esc_attr( $id ); ?>_other]" 
-						value="<?php echo esc_attr( $options[ $id . '_other' ] ?? '' ); ?>" 
+						type="text"
+						id="aspireupdate-settings-field-<?php echo esc_attr( $id ); ?>_other"
+						name="<?php echo esc_attr( $this->option_name ); ?>[<?php echo esc_attr( $id ); ?>_other]"
+						value="<?php echo esc_attr( $options[ $id . '_other' ] ?? '' ); ?>"
```

Comment:
🤔 What change am I not seeing?

Reply:
As above, my VSCode settings removed the extra space at the end of the line automatically.